style.css

```diff
@@ -0,0 +1,81 @@
+ul.header {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+}
+ul.header li {
+    float: left;
+}
+
+ul.header li a {
+    display: block;
+    color: white;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
+ul.header li a:hover {
+    background-color: #111;
```

Comment:
- Kindly request you also to use RGB

Reply:
- [ ] The same thing here. It would be better to include an explanation.

Reply:
- [ ] 🤜 Great job for using `background-color` for your page. I suggest you use **RGB** instead of **hex** color representation because it will rise **CSS** linter errors.